src/incubator/expandableOverlay/index.tsx

```diff
@@ -86,6 +86,9 @@ const ExpandableOverlay = (props: ExpandableOverlayProps, ref: any) => {
     toggleExpandable
   }));
 
+  const modalTopBar = useMemo(() => showTopBar && <Modal.TopBar onDone={closeExpandable} {...topBarProps}/>,
```

Comment:
Why no simply call it "topBar"? It is passed to both Modal and Dialog

Reply:
Yes it is passed in both cases if `showTopBar` is set to `true`